aws-feed/operations.py

```diff
@@ -0,0 +1,59 @@
+""" Copyright start
+  Copyright (C) 2008 - 2022 Fortinet Inc.
+  All rights reserved.
+  FORTINET CONFIDENTIAL & FORTINET PROPRIETARY SOURCE CODE
+  Copyright end """
+
+import requests
+from connectors.core.connector import get_logger, ConnectorError
+
+logger = get_logger('aws-feed')
+
+
+def check_health(config):
+    try:
+        return get_indicators(config, {}, True)
```

Comment:
can we make this a little lightweight by skipping parsing the response. we can just look at some status code and top level keys in the response

Reply:
handled in get_indicators() actions.